var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -41,16 +42,25 @@ class Mpich(AutotoolsPackage):
     version('3.1.1', '40dc408b1e03cc36d80209baaa2d32b7')
     version('3.1',   '5643dd176499bfb7d25079aaff25f2ec')
     version('3.0.4', '9c5d5d4fe1e17dd12153f40bc5b6dbc0')
+    version('develop', git='git://github.com/pmodels/mpich')
 
     variant('hydra', default=True,  description='Build the hydra process manager')
     variant('pmi',   default=True,  description='Build with PMI support')
     variant('romio', default=True,  description='Enable ROMIO MPI I/O implementation')
     variant('verbs', default=False, description='Build support for OpenFabrics verbs.')
+    variant('ch4-ofi', default=False, description='Build ch4 device with OFI (libfabric) netmod.')
```

Comment:
maybe a multivalued variant is a better option for this?

Reply:
i.e. `ofi`, `ucx`, `none` (use `ch3`)

Reply:
i don't think it's documented, but look at `openmpi` 's variants.

Reply:
Sure, will take a look.